ggshield/core/text_utils.py

```diff
@@ -123,8 +123,9 @@ def display_error(msg: str) -> None:
     click.echo(format_text(msg, STYLE["error"]), err=True)
 
 
-def display_warning(msg: str) -> None:
-    click.echo(format_text(msg, STYLE["warning"]), err=True)
+def display_warning(msg: str, critical: bool = False) -> None:
```

Comment:
I am not fond of this because I am pretty sure the API change is going to stay there after we remove the deprecation warning. I think it's simpler to switch to `display_error()` instead (with a comment explaining why we use it there), even if it's not an error.

Reply:
I was hesitant of using `display_error()` because we were displaying a warning and not an error. But I agree, it's enough for the task and we can add a comment to explain why we use it.

Thanks for the review :pray: